### DISCUSSION TITLE
🧩 FEATURE REQUEST: sort results in search

### DISCUSSION THREAD

**Discussion by anuraglol** [dailydotdev/daily#457] [Feature Requests]
Users should be allowed to sort the search results, like by date, number of upvotes, etc. when they search for a topic

**Comment by nimrodkra** [comment]
Hi @kr-anurag thanks for the feedback! That's definitely something we plan to build. There's still no specific release date but I'll keep you posted once it's out.

**Comment by idoshamun** [comment]
Let's keep the convo here: https://github.com/dailydotdev/daily/discussions/294
